prober/derp.go

```diff
@@ -289,9 +319,12 @@ func (d *derpProber) updateMap(ctx context.Context) error {
 	return nil
 }
 
-func (d *derpProber) ProbeUDP(ipaddr string, port int) ProbeFunc {
-	return func(ctx context.Context) error {
-		return derpProbeUDP(ctx, ipaddr, port)
+func (d *derpProber) ProbeUDP(ipaddr string, port int) ProbeClass {
```

Comment:
considering the string further down is `derp_udp` should this be `ProbeDerpUDP`?

Reply:
It's already a "method" on derpProber, so perhaps we don't need another "Derp" in there.